app/models/metric/processing.rb

```diff
@@ -90,7 +90,7 @@ def self.process_derived_columns(obj, attrs, ts = nil)
       when "numvcpus" # This is actually logical cpus.  See note above.
         # Do not derive "available" values if there haven't been any usage
         # values collected
-        result[col] = state.numvcpus if obj.kind_of?(VmOrTemplate) && have_cpu_metrics && state.numvcpus.to_i > 0
+        result[col] = state.numvcpus if have_cpu_metrics && state.numvcpus.to_i > 0
```

Comment:
Its possible this should have a been check for `respond_to?(:numvcpus)`...we might need that instead of removing the VmOrTemplate conditional entirely...not really sure why the conditional was there in the first place except maybe to prevent things like Storage metrics from blowing up.